### PR TITLE
Add GameModeOnlyParameter for allow disabe image effect apply in SceneView

### PR DESCRIPTION
**Pull request by edwin0cheng**
This PR add a new GameModeOnlyParameter in RenderWithMaterial which allow user disable/enable the image effect on SceneView.

### CODE REVIEW

File: Assets/ImageEffectGraph/Runtime/PostProcessing/GameModeOnlyParameters.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering.PostProcessing;
+
+namespace ImageEffectGraph.PostProcessing
+{
+    [Serializable]
+    public sealed class GameModeOnlyParameter : ParameterOverride<bool>
```
Comment (iBicha) [inline Assets/ImageEffectGraph/Runtime/PostProcessing/GameModeOnlyParameters.cs]: Is there a particular reason this class is needed? It looks to me like a `BoolParameter` would do as well without adding another class.

Comment (edwin0cheng) [reply Assets/ImageEffectGraph/Runtime/PostProcessing/GameModeOnlyParameters.cs]: It is because I don't know they have this class :)
Fixed.

Comment (iBicha) [reply Assets/ImageEffectGraph/Runtime/PostProcessing/GameModeOnlyParameters.cs]: Very well!

**Comment by iBicha** [comment]
Awesome. Thank you for your contribution!

**Pull request merged**